external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,560 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
+| `from_template`     | Use a [column template](templates.md) (Optional)                                                                                                                                                                                                                                                | *none*                                                                                          |
+| `label`             | The title of the column (Optional)                                                                                                                                                                                                                                                              | The capital case version of the column header in the csv file with spaces replacing underscores |
+| `description`       | A description of the contents of the column (Optional)                                                                                                                                                                                                                                          | *none*                                                                                          |
+| `from_existing`     | The uri of the resource for reuse/extension (Optional)                                                                                                                                                                                                                                          | *none*                                                                                          |
+| `definition_uri`    | A uri of a resource to show how the column is created/managed (e.g. a uri of a PDF explaining a list of units) (Optional)                                                                                                                                                                       | *none*                                                                                          |
+| `cell_uri_template` | **(Advanced)** Override the uri generated for values within the uri (Optional)                                                                                                                                                                                                                  | *none*                                                                                          |
+| `code_list`         | Generate a code-list (true), suppress a code-list (false), file path to a [code-list-config.json](code-list-config.md#defining-a-code-list-configuration-file), [in-line code list](code-list-config.md#defining-an-in-line-code-list) (json), or link to an externally-defined code list (uri) | true                                                                                            |
```

Comment:
Can we order these tables so the advanced stuff is at the bottom? i.e. Most useful stuff first, least useful stuff last.